src/libs/PersonalDetailsUtils.ts

```diff
@@ -0,0 +1,170 @@
+import Onyx, {OnyxEntry} from 'react-native-onyx';
+import ONYXKEYS from '../ONYXKEYS';
+import * as Localize from './Localize';
+import * as UserUtils from './UserUtils';
+import * as LocalePhoneNumber from './LocalePhoneNumber';
+import * as OnyxTypes from '../types/onyx';
+
+type PersonalDetailsList = Record<string, OnyxEntry<OnyxTypes.PersonalDetails>>;
```

Comment:
```suggestion
type PersonalDetailsList = Record<string, OnyxTypes.PersonalDetails | null>;
```

In this case would make more sense not using `OnyxEntry` internally in this way.

BTW, do you think we can move this `PersonalDetailsList` type to `src/types/onyx/PersonalDetails.ts` and use it in [ONYXKEYS.ts](https://github.com/Expensify/App/blob/main/src/ONYXKEYS.ts#L327) as well?